code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -121,6 +122,10 @@
 		stun *= 0.5
 		if(status)		//Checks to see if the stunbaton is on.
 			agony *= 0.5	//whacking someone causes a much poorer contact than prodding them.
+			if(sheathed) //however breaking the skin results in a more potent electric shock or some bullshit. im a coder, not a doctor
```

Comment:
`whacking them produces a poorer contact, but a better contact`

Reply:
im a coder not a doctor, and i can't take responsibility for the stupidity of those who came before me. some quick pseudoscience:

a direct, non-breaking contact has better distribution of electromagic that allows the target to be more routinely stunned.

contact that breaks the skin is less uniform and less localized, but distributes across the entire body's interior 'n shit. 

fuck, players can't even read these comments, i ain't gotta explain shit.

Reply:
>fuck, players can't even read these comments, i ain't gotta explain shit.

we can and you do :)